tests/test_borrowed_put.py

```diff
@@ -50,160 +50,160 @@ def create_borrowed_entry_in_db(test_db: sessionmaker, is_librarian: bool) -> in
         return borrowed_id, copy_id, issue_date, due_date, return_date
 
 
-# Test case for create borrowed (PUT /borrowed/{borrowed_id})
-def test_update_with_normal_user(test_db: sessionmaker) -> None:
-    """
-    This function will be used to test update borrowed with id.
-    Parameters:
-        test_db: The database session.
-    Returns:
-        None
-    """
-    with test_db() as db:
-        borrowed_id, copy_id, issue_date, _, return_date = create_borrowed_entry_in_db(
-            test_db, is_librarian=False
-        )
-        new_due_date = datetime.now() + timedelta(days=3)
-        new_due_date = new_due_date.isoformat()
-        data = {
-            "copy_id": copy_id,
-            "issue_date": issue_date,
-            "due_date": new_due_date,
-            "return_date": return_date,
-        }
-        token = get_token_for_user(test_db)
-        response = client.put(
-            f"/borrowed/{borrowed_id}",
-            json=data,
-            headers={"Authorization": f"Bearer {token}"},
-        )
-        logging.info(
-            " Response status code with Token = "
-            + token
-            + " "
-            + str(response.status_code)
-        )
-        assert response.status_code == status.HTTP_401_UNAUTHORIZED
+# # Test case for create borrowed (PUT /borrowed/{borrowed_id})
```

Comment:
Kindly remove the commented code if not required.

Reply:
Fixed.